src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml

```diff
@@ -61,7 +61,11 @@
                 </Grid.ColumnDefinitions>
```

Comment:
All changes to the AccessibilityInsights and SharedUx projects are scoped to adding AutomationIDs.